packages/story-editor/src/components/panels/design/captions/captions.js

```diff
@@ -124,11 +124,11 @@ function CaptionsPanel({ selectedElements, pushUpdate }) {
   );
 
   const handleChangeTrack = useCallback(
-    (attachment) => {
+    (resource) => {
       const newTracks = {
-        track: attachment?.url,
-        trackId: attachment?.id,
-        trackName: attachment?.filename,
+        track: resource?.src,
+        trackId: resource?.id,
+        trackName: resource?.filename,
```

Comment:
Is this the reason for adding `filename` to resource everywhere?

Reply:
Yes

Reply:
Done in https://github.com/rtCamp/web-stories-wp/pull/731/commits/acb8941c86df08e0fb3801658f7077fcbe56538f